scripts/conflicts/diff.js

```diff
@@ -22,35 +28,94 @@ console.log(`Generating diffs for conflicts on ${server}`);
 
 const DB = new PouchDB(server);
 
-DB.query('medic-conflicts/conflicts')
+const writeToFile = function(path, content) {
+  content  = JSON.stringify(content, null, 3); 
+  fs.writeFile(path, content, (err) => {
```

Comment:
This is a callback function. Please either: 
- use `fs.promises.writeFile` and make the function async
- keep the callback but make everything callback based. 
- make everything synchronous `fs.writeFileSync`

```javascript
const writeToFile = async (path, content) => {
  try {
    const jsonContent = JSON.stringify(content, null, 3);
    await fs.promises.writeFile(path, content);
  } catch (err) {
    console.error('Error when writing file', err);
    throw err; 
  }
}
```